odl/space/cu_ntuples.py

```diff
@@ -668,6 +676,39 @@ def exponent(self):
         """Exponent of the norm and distance."""
         return self._space_funcs.exponent
 
+    @property
+    def weighting(self):
```

Comment:
I'll check. I *think* it's also present in `DiscreteLp`, but it would actually make sense in `Discretization` directly.